### ISSUE TITLE
Impossible to select one particular fighting art at the bottom of the board

### ISSUE THREAD

**Issue by irishwizz** [maximo1984/Kingdom-Death-Maximo-Kijan-Mod#53]
Hi,

In the new campaign, I am in my second year. One of my characters rolled and got "orator of death" fighting art... but I can't select a particular fighting art. When I click on fighting arts, and then on "random" and type the name of my fighting art before clicking "Draw", it does not draw my fighting art but the name of my fighting art gets replaced with "random"

**Comment by KarateSnoopy** [comment]
Type "Orator of Death" in the box and click draw.  Its case sensitive.  I'll turn this into a bug that says "Draw" shouldn't be case sensitive as its annoying :)

**Comment by irishwizz** [comment]
No, even with the capital letters at the right place, it did not work, but I found why:
if you press enter after typing the name (thinking it will validate), it does a return carriage and invalidate the name. Could it be possible to forbid the return carriage in this box?
